api/base/serializers.py

```diff
@@ -279,6 +299,53 @@ def to_representation(self, data):
             self.child.to_representation(item, envelope=None) for item in data
         ]
 
+    # Overrides ListSerializer which doesn't support multiple update by default
+    def update(self, instance, validated_data):
+        if len(instance) != len(validated_data):
+            raise exceptions.NotFound()
+        instance_mapping = {item._id: item for item in instance}
+        data_mapping = {item.get('_id', None): item for item in validated_data}
+
+        ret = []
+
+        for resource_id, data in data_mapping.items():
+            resource = instance_mapping.get(resource_id, None)
+            ret.append(self.child.update(resource, data))
+
+        return ret
+
+    # overrides ListSerializer
+    def run_validation(self, data=empty):
+        meta = getattr(self, 'Meta', None)
+        bulk_limit = getattr(meta, 'bulk_limit', self.DEFAULT_BULK_LIMIT)
+
+        num_items = len(data)
```

Comment:
If `data == empty`, the length check will fail. (`empty` is a blank class definition, not an iterable)